packages/suite-desktop-core/e2e/tests/analytics/events.test.ts

```
@@ -9,8 +9,7 @@ test.describe('Analytics Events', { tag: ['@group=suite', '@webOnly'] }, () => {
         await onboardingPage.disableNecessaryFirmwareChecks();
     });
 
-    //TODO: Fix instable test
-    test.skip('reports transport-type, suite-ready and device-connect/device-disconnect events when analytics is initialized and enabled', async ({
+    test.only('reports transport-type, suite-ready and device-connect/device-disconnect events when analytics is initialized and enabled', async ({
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)